src/lib/hokusai.py

```diff
@@ -3,8 +3,7 @@
 from lib.util import run_cmd
 
 
-def env_unset(project_dir, env, var_names):
-  ''' run hokusai env unset for vars for project '''
-  for var in var_names:
-    logging.debug(f'unsetting {var}...')
-    run_cmd(f'hokusai {env} env unset {var}', project_dir)
+def env_unset(project_dir, env, var):
+  ''' run hokusai env unset for var for project '''
+  logging.debug(f'unsetting {var}...')
+  run_cmd(f'hokusai {env} env unset {var}', project_dir)
```

Comment:
```suggestion
  run_cmd(f'hokusai {env} env unset {var}', {project_dir})
```

Reply:
`project_dir` is for [2nd arg](https://github.com/artsy/opstools/blob/daac010c034c2126d2756b08d7adf35572288458/src/lib/util.py#L64). should be okay w/o curly braces?

Reply:
oh woops I misread this as being a full string! yes ignore my comment 😅